### ISSUE TITLE
Audit and update pages to use consistent TTY: 711 links

### ISSUE THREAD

**Issue by RLHecht** [department-of-veterans-affairs/va.gov-team#40831]
## Issue Description
We have updated the style guide entry for how TTY numbers should be coded so that the TTY is included in the link. Currently this isn't consistent on our pages. We need to do an audit to find all our pages where we use TTY numbers and update these to match our style guide.
Here's an example of how it should look:

![Screenshot (10)](https://user-images.githubusercontent.com/102090263/194420758-87e98a08-efe0-4469-9601-12ea3c8e80c5.png)

## Tasks
- [x] Reach out to CMS team and PW to determine who can help with audit
- [ ] Update pages to correct style
- [ ] Post a list of pages affected in this ticket 
- [ ] Review and publish

## Acceptance Criteria
- [ ] All TTY links are formatted the same across the site

**Comment by aliyahblackmore** [comment]
Update 8/5: I started on this today and will continue next week!

**Comment by aliyahblackmore** [comment]
Update 8/25: Moving back to this ticket. Also adding that Danielle confirmed that we should remove "If you have hearing loss..." and include "(TTY:711)" in parenthesis immediately after the phone number.

**Comment by aliyahblackmore** [comment]
Hi @megzehn ! I went through the first 54 entries in the [audit doc](https://github.com/department-of-veterans-affairs/va.gov-team/issues/40831#issuecomment-1172493256) and I'm attaching the pages that I've reviewed so far. I'll continue to go through this today,

**Spreadsheet document**
[711_Audit_Changed Pages_AB_Copy.xlsx](https://github.com/department-of-veterans-affairs/va.gov-team/files/9662214/711_Audit_Changed.Pages_AB_Copy.xlsx)

Of these 54 entries, there were a few that are on pages that will be going live soon. I made edits to a few of them yesterday (pinged Beth/Katherine) and though these are small changes to 711, I will just ping writers on other pages as a heads up before I edit in Drupal; I will come back to those.

This spreadsheet also includes notes on any 711 changes where they varied from the style guide (for example, federal relay services listed on VA pages) and small style/content catches that I noticed (and briefly added as a note on the line).

Notes/Clarifications:
- In addition to being sure the entire "TTY: 711" is linked, we are also supposed to remove any instance of "If you have hearing loss..." and move the TTY next to the telephone number.
- As I note in the spreadsheet, there are a few pages that have multiple telephone numbers. As Beth stated, I included TTY only when numbers were spaced out and if they were in different accordions. For telephone numbers that may have been the same, again, I included TTY: 711 only if they were not close to each other on the page and in different accordions.

**Comment by coforma-jamie** [comment]
> [!NOTE]
> ## Accessibility Digital Experience (DE) Ticket Status Update
> Accessibility DE is OCTO’s new accessibility team. We’re working on a new intake process. In the meanwhile:
> - This ticket has been moved to validate and will be closed on the Accessibility DE pipeline (only).
> - This ticket will remain “open” until the work from content and IA is completed.
> 
> If you need additional accessibility support, contact the Accessibility DE team on [#accessibility-help](https://dsva.slack.com/archives/C8E985R32).
> **Thank you for working with the accessibility team!**
